eng/Versions.props

```diff
@@ -125,8 +125,10 @@
     <MonoAddinsPackageVersion>1.3.8</MonoAddinsPackageVersion>
     <MonoDevelopSdkPackageVersion>1.0.15</MonoDevelopSdkPackageVersion>
     <MoqPackageVersion>4.16.0</MoqPackageVersion>
+    <NerdbankStreamsPackageVersion>2.6.81</NerdbankStreamsPackageVersion>
     <OmniSharpExtensionsLanguageServerPackageVersion>0.18.1</OmniSharpExtensionsLanguageServerPackageVersion>
     <OmniSharpMSBuildPackageVersion>1.37.13</OmniSharpMSBuildPackageVersion>
+    <StreamJsonRpcPackageVersion>2.8.21</StreamJsonRpcPackageVersion>
```

Comment:
cc/ @dotnet/razor-tooling this'll likely require a co-ordinated insertion for the `StreamJsonRpc` upgrade.

Reply:
> cc/ @dotnet/razor-tooling this'll likely require a co-ordinated insertion for the `StreamJsonRpc` upgrade.

Coordinated with WebTools?

Reply:
Yep!